rust/stub/src/unified_sections.rs

```diff
@@ -0,0 +1,42 @@
+/// List of PE sections that have a special meaning with respect to
+/// UKI specification.
+/// This is the canonical order in which they are measured into TPM
+/// PCR 11.
+/// !!! DO NOT REORDER !!!
+pub enum UnifiedSection {
+    Linux,
+    OsRel,
+    CmdLine,
+    Initrd,
+    Splash,
+    DTB,
+    PcrSig,
+    PcrPkey
+}
```

Comment:
```suggestion
#[repr(u8)]
pub enum UnifiedSection {
    Linux = 0,
    OsRel = 1,
    CmdLine = 2,
    Initrd = 3,
    Splash = 4,
    DTB = 5,
    PcrSig = 6,
    PcrPkey = 7,
}
```

This way it's not as easy to get the values wrong by just changing the ordering

Reply:
Oh I see your point now